aa-sdk/core/src/errors/client.ts

```diff
@@ -88,3 +88,19 @@ export class InvalidNonceKeyError extends BaseError {
     );
   }
 }
+
+/**
+ * Error class denoting that the provided entity id is invalid because it's overriding the native entity id.
+ */
+export class EntityIdOverrideError extends BaseError {
+  override name = "InvalidNonceKeyError";
```

Comment:
The class name and string `name` don't match. I'm guessing the string is wrong?

Reply:
Addressed in [fb6d3b6](https://github.com/alchemyplatform/aa-sdk/pull/1232/commits/fb6d3b69e2ffc5dac48ffc8f80f0a5d83a3454d3)!